### ISSUE TITLE
Wallets

### ISSUE THREAD

**Issue by llorllale** [zold-io/java-api#4]
Let's implement `Wallets`:

https://github.com/zold-io/java-api/blob/f323e05f461a45ebe4a8b791abbc567be7165f8e/src/main/java/io/zold/api/Wallets.java#L34-L40

Let's name the implementation `WalletsIn`.

**Comment by llorllale** [comment]
@0crat hey

**Comment by 0crat** [comment]
> @0crat hey ([here](https://github.com/zold-io/java-api/issues/4#issuecomment-400075284))

@llorllale Hey, what's up, how is it going?

**Comment by llorllale** [comment]
@0crat in

**Comment by 0crat** [comment]
> @0crat in ([here](https://github.com/zold-io/java-api/issues/4#issuecomment-400075994))

@llorllale Job [#4](https://github.com/zold-io/java-api/issues/4) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by llorllale** [comment]
@0crat assign @t-izbassar

**Comment by 0crat** [comment]
> @0crat assign @t-izbassar ([here](https://github.com/zold-io/java-api/issues/4#issuecomment-402118561))

@llorllale The job [#4](https://github.com/zold-io/java-api/issues/4) assigned to @t-izbassar[/z](https://www.0crat.com/u/t-izbassar), here is [why](https://www.0crat.com/footprint/CAZUREFND/1841123804); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
The architect of the project has changed; @yegor256[/z](https://www.0crat.com/u/yegor256) is not at this role anymore; @llorllale[/z](https://www.0crat.com/u/llorllale) is the architect now

**Comment by 0pdd** [comment]
@llorllale 2 puzzles [#12](https://github.com/zold-io/java-api/issues/12), [#13](https://github.com/zold-io/java-api/issues/13) are still not solved.

**Comment by t-izbassar** [comment]
@llorllale this is done by #10. Can we close it?

**Comment by llorllale** [comment]
@t-izbassar thanks!

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@7029763d-567a-43e6-8426-2f46b4b74955, thread: pool-370-thread-1
com.zerocracy.pm.Claims[122] java.lang.IllegalStateException: Duplicate claims are not allowed in [CAZUREFND](https://www.0crat.com/p/CAZUREFND), can't add this XML to 2 existing ones (Close job;{job=gh:zold-io/java-api#4, reason=GitHub issue was closed},Close job;{job=gh:zold-io/java-api#4, reason=GitHub issue was closed}):
<?xml version="1.0" encoding="UTF-8"?>
<claim id="1851212139">
   <created>2018-07-04T12:09:12.775Z</created>
   <type>Close job</type>
   <token>github;zold-io/java-api;4</token>
   <author>llorllale</author>
   <params>
      <param name="job">gh:zold-io/java-api#4</param>
      <param name="reason">GitHub issue was closed</param>
   </params>
   <until>2018-07-04T12:24:12.777Z</until>
</claim>

```

[0.24.10](https://github.com/zerocracy/farm): Issue: zold-io/java-api#4, Action: `closed`

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +38 point(s) just awarded to @t-izbassar[/z](https://www.0crat.com/u/t-izbassar)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by 0pdd** [comment]
@llorllale the puzzle [#12](https://github.com/zold-io/java-api/issues/12) is still not solved; solved: [#13](https://github.com/zold-io/java-api/issues/13).

**Comment by 0pdd** [comment]
@llorllale 2 puzzles [#65](https://github.com/zold-io/java-api/issues/65), [#66](https://github.com/zold-io/java-api/issues/66) are still not solved; solved: [#12](https://github.com/zold-io/java-api/issues/12), [#13](https://github.com/zold-io/java-api/issues/13).

**Comment by 0pdd** [comment]
@llorllale the puzzle [#65](https://github.com/zold-io/java-api/issues/65) is still not solved; solved: [#12](https://github.com/zold-io/java-api/issues/12), [#13](https://github.com/zold-io/java-api/issues/13), [#66](https://github.com/zold-io/java-api/issues/66).

**Comment by 0pdd** [comment]
@llorllale 2 puzzles [#85](https://github.com/zold-io/java-api/issues/85), [#86](https://github.com/zold-io/java-api/issues/86) are still not solved; solved: [#12](https://github.com/zold-io/java-api/issues/12), [#13](https://github.com/zold-io/java-api/issues/13), [#65](https://github.com/zold-io/java-api/issues/65), [#66](https://github.com/zold-io/java-api/issues/66).
